google/cloud/bigtable/examples/opencensus/bigtable_opencensus.cc

```diff
@@ -0,0 +1,166 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! [all code]
+
+//! [bigtable includes]
+#include "google/cloud/bigtable/table.h"
+#include "google/cloud/bigtable/table_admin.h"
+//! [bigtable includes]
+
+#ifndef OPENCENSUS_DISABLED
+#include "opencensus/exporters/stats/stackdriver/stackdriver_exporter.h"
+#include "opencensus/exporters/stats/stdout/stdout_exporter.h"
+#include "opencensus/exporters/trace/stackdriver/stackdriver_exporter.h"
+#include "opencensus/exporters/trace/stdout/stdout_exporter.h"
+#include "opencensus/stats/stats.h"
+#include "opencensus/trace/sampler.h"
+#include "opencensus/trace/trace_config.h"
+#include <grpcpp/opencensus.h>
+#endif  // OPENCENSUS_DISABLED
+
+int main(int argc, char* argv[]) try {
+  if (argc != 4) {
+    std::string const cmd = argv[0];
+    auto last_slash = std::string(cmd).find_last_of('/');
+    std::cerr << "Usage: " << cmd.substr(last_slash + 1)
+              << " <project_id> <instance_id> <table_id>" << std::endl;
+    return 1;
+  }
+
+  std::string const project_id = argv[1];
+  std::string const instance_id = argv[2];
+  std::string const table_id = argv[3];
+
+#ifndef OPENCENSUS_DISABLED
```

Comment:
We need a README.md or a `*.dox` file that explains:

- How to setup stackdriver to read the open census data, just a link to the right documentation on the stackdriver and/or opencensus page would be enough.
- How to see the results of this example in the stack driver pages.

Reply:
included README.md